Bottom: e7a0a83f215a9054d2f686dce4f8c9c6790dd8d4
Top:    99254e64fda553d6b67ea10b083b9e95b8d1f311
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-03 17:19:46 +0000

Refresh of rename-node_label-and

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 37c25f8..0d5183d 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -372,11 +372,11 @@ node_add_single_point (VNODE * a, Vector p)
 }				/* node_add_point */
 
 /*
-node_label
+edge_label
  (C) 2006 harry eaton
 */
 static unsigned int
-node_label (VNODE * pn)
+edge_label (VNODE * pn)
 {
   CVCList *first_l, *l;
   char this_poly;
@@ -458,7 +458,7 @@ node_label (VNODE * pn)
   if (region == SHARED || region == SHARED2)
     return UNKNWN;
   return region;
-}				/* node_label */
+}				/* edge_label */
 
 /*
  add_descriptors
@@ -1093,7 +1093,7 @@ label_contour (PLINE * a)
     {
       if (cur->cvc_next)	/* examine cross vertex */
 	{
-	  label = node_label (cur);
+	  label = edge_label (cur);
 	  if (first_labelled == NULL)
 	    first_labelled = cur;
 	  continue;
